_data/learning.yml

```
@@ -119,6 +119,11 @@ reference-courses:
   - name: "PRACE Course: Advanced Fortran Topics"
     url: https://doku.lrz.de/display/PUBLIC/PRACE+Course%3A+Advanced+Fortran+Topics
 
+  - name: "The 'F' Word"
```

Let's use the wording from here:

https://twitter.com/fortranlang/status/1487504884865904644

```suggestion
  - name: "The 'F' Word - Programming in Fortran"
```

---

@certik -  I think your suggestion is a good one. My intention is to reflect the observation that computer scientists are often offended (in some way) about developing new applications in Fortran these days. Adding the clarifier "Programming in Fortran" definitely helps affirm which offensive 'F' word I mean.